### PR TITLE
Fixing exception handling and interface usage for siddhi try it ui component

### PR DESCRIPTION
**Pull request by thiliA**

### CODE REVIEW

File: components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java [hunk 1]
```diff
@@ -55,6 +56,8 @@
     private static Log log = LogFactory.getLog(SiddhiTryItClient.class);
     private static Gson gson = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
     private SimpleDateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS");
+    Pattern eventPattern = Pattern.compile("(\\S+)=\\[(.*)\\]");
```
Comment (prabathabey) [inline components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java]: Missing access modifers

File: components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java [hunk 2]
```diff
@@ -55,6 +56,8 @@
...
+    Pattern delayPattern = Pattern.compile("(delay\\()(\\d+)+");
```
Comment (prabathabey) [inline components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java]: Missing access modifiers.

File: components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java (hunk 1 above)
Comment (prabathabey) [inline components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java]: To further improve this, you can make this a static variable too to further improve memory efficiency. Might not get you any significant results, but still we should be able to adopt this as a best practice.

File: components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java (hunk 2 above)
Comment (prabathabey) [inline components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java]: Same explained above, goes here as well.

File: components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java (hunk 1 above)
Comment (thiliA) [inline components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java]: @prabathabey 
Thanks a lot for the suggestions. Fixed in commit d2855f5

File: components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java (hunk 2 above)
Comment (thiliA) [inline components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java]: @prabathabey
Thanks a lot for the suggestions. Fixed in commit d2855f5

**Pull request merged**